src/player-statistics.js

```diff
@@ -255,7 +260,7 @@ export default [
   {group: 'Gemstore', title: 'Permanent Contracts', key: 'statistics.permanentContracts', sprite: 'permanent-contract', gold: false, by_playtime: true, decimals: 0},
   {group: 'Gemstore', title: 'Toys', key: 'statistics.gemstoreToys', sprite: 'gemstore-toys', gold: false, by_playtime: true, decimals: 0},
   {group: 'Gemstore', title: 'Shared Inventory Slots', key: 'statistics.sharedInventorySlots', sprite: 'shared-inventory-slot', gold: false, by_playtime: true, decimals: 0},
-  {group: 'Gemstore', title: 'Permanent Lounge Passes', key: 'statistics.permanentLoungePasses', sprite: 'permanentLoungePasses', gold: false, by_playtime: true, decimals: 0},
+  {group: 'Gemstore', title: 'Permanent Lounge Passes', key: 'statistics.UniquePermanentLoungePasses', sprite: 'permanentLoungePasses', gold: false, by_playtime: true, decimals: 0},
```

Comment:
This broke again.

Reply:
This is insane..and i'll be asking your help on this. :(